src/eynollah/eynollah.py

```diff
@@ -24,7 +24,6 @@
 import json
 
 from loky import ProcessPoolExecutor
-from PIL.Image import Image
```

Comment:
Indeed – thanks for spotting!

Could you please show a recipe how to reproduce the error (so we can perhaps cover it in a test)?

To explain: this import actually was made (by @kba, for type hints) before your `resize_image` changes imported PIL, but  recently came out of use when I refactored the class, so the type annotation went away.

Reply:
@bertsky This issue occurs when using the 'Eynollah ocr' with the RNN-CNN model (as of now), specifically when trying to draw predicted text on an image to visually check the predictions (-doit dir_out -dtoi ). The error happens with the command:
image_text = Image.new("RGB", (img.shape[1], img.shape[0]), "white")
and the message says: "Image has no feature named new."

Reply:
`PIL.Image` is the module (which has a `new` function), `PIL.Image.Image` is the class that is instantiated by `PIL.Image.new`). We should of course broaden the coverage so we spot such errors, but AFAICT this particular problem should be fixed with https://github.com/qurator-spk/eynollah/pull/151/commits/e2907f67e0bab786b8c810cdb45caf36dc9eefa6.

Reply:
Thanks @vahidrezanezhad!

Ok, so far the OCR is not covered by tests at all (neither as `ocr=True` in layout nor via extra command/class).

If you show me a typicall call and what to test for (certain words in the result, or just successful run), I can add that.

Reply:
No, it doesn't require any text or OCR to reproduce the issue. I'm simply trying to create a white image to draw text on, and the error occurs at that point. You can reproduce the same error by running the following two lines:

`from PIL.Image import Image
image_next = Image.new('RGB', (100, 100), 'white')`

Reply:
@vahidrezanezhad 

I know, I understand – again, the import thing is trivial. My issue is: we should try to test-cover what you are developing. (See esp. the coverage [summaries](https://github.com/bertsky/eynollah/actions/runs/14286412468#summary-40042008259) and [reports](https://github.com/bertsky/eynollah/actions/runs/14286412468/artifacts/2888663094) – we are still far away from 100%.)

Hence my question how to set up a new test for the OCR feature.